plugins/router/template/router.go

```diff
@@ -144,6 +145,20 @@ func newTemplateRouter(cfg templateRouterCfg) (*templateRouter, error) {
 	return router, nil
 }
 
+func endpointsForAlias(alias ServiceAliasConfig, svc ServiceUnit) []Endpoint {
+	if len(alias.PreferPort) == 0 {
+		return svc.EndpointTable
+	}
+	endpoints := make([]Endpoint, 0, len(svc.EndpointTable))
+	for i := range svc.EndpointTable {
+		endpoint := svc.EndpointTable[i]
+		if endpoint.PortName == alias.PreferPort || endpoint.Port == alias.PreferPort {
```

Comment:
Catching up this week. Should be easy to add multiple ports in here, no? 
Let the ```ServiceAliasConfig``` have a list of preferred ports and 
add 'em to a map[string]bool ```mapR``` here and then later just change this check: 
```endpoint.PortName == alias.PreferPort || endpoint.Port == alias.PreferPort``` 
to ```mapR[endpoint.PortName]  || mapR[endpoint.Port]``` or something to that effect.